markdown/ch6.markdown

```diff
@@ -63,35 +74,19 @@ We definitely can't fit all of the linear algebra theory needed to understand
 the properties of this matrix, but suffice it to say: it has some *great*
 properties. We will exploit a couple in the following paragraphs.
 
-For example, a common problem in network analysis is visualization. How do you
-draw nodes and links in such a way that you don't get a complete mess such as
-this one?
-
-![network hairball](https://upload.wikimedia.org/wikipedia/commons/9/90/Visualization_of_wiki_structure_using_prefuse_visualization_package.png)
-**Graph created by Chris Davis. [CC-BY-SA-3.0](https://commons.wikimedia.org/wiki/GNU_Free_Documentation_License).**
-**[Ed note: my reading is that we can use this figure as long as we include the
-copyright notice and don't put DRM on the books, which O'Reilly doesn't do
-anyway.]**
-
-One way is to put nodes that share many links close together, and it turns out
-that this can be done by using the second-smallest eigenvalue of the Laplacian
-matrix, and its corresponding eigenvector, which is so important it has its
-own name: the
-[Fiedler vector](https://en.wikipedia.org/wiki/Algebraic_connectivity#The_Fiedler_vector).
-
-As a quick aside, an eigenvector $v$ of a matrix $M$ is a vector that
+First, we will look at the *eigenvectors* of $L$.
+An eigenvector $v$ of a matrix $M$ is a vector that
 satisfies the property $Mv = \lambda v$ for some number $\lambda$,
 known as the eigenvalue.  In other words, $v$ is a special vector in
-relation to $M$ because $Mv$ simply scales the vector, without
-changing its direction.[^eigv_example]
+relation to $M$ because $Mv$ simply changes the size of the vector, without
+changing its direction. As we will soon see, eigenvectors have many useful
+properties — sometimes seeming even magical!
 
-[^eigv_example]: As an example, consider a 3x3 rotation matrix $R$
-                 that, when multiplied by any 3-dimensional vector
-                 $p$, rotates it $30^\circ$ degrees around the z-axis.
-                 $R$ will rotate all vectors except for those that lie
-                 *on* the z-axis.  For those, we'll see no effect, or
-                 $Rp = p$, i.e. $Rp = \lambda p$ with eigenvalue
-                 $\lambda = 1$.
+As an example, a 3x3 rotation matrix $R$, when multiplied by any
+3-dimensional vector $p$, rotates it $30^\circ$ degrees around the z-axis.  $R$
+will rotate all vectors except for those that lie *on* the z-axis.  For those,
+we'll see no effect, or $Rp = p$, i.e. $Rp = \lambda p$ with eigenvalue
+$\lambda = 1$.
```

Comment:
Would it be possible to show this with a plot, or something? I look at the exercise below, and wonder how I'm going to be able to tell if my matrix has been rotated 😛